### DISCUSSION TITLE
Scoreboard During the playoffs?

### DISCUSSION THREAD

**Discussion by drmuhlestein** [riffnshred/nhl-led-scoreboard#456] [General]
It looks like the scoreboard stopped working at midnight last night. I am assuming it is because the season is over. 
Does anyone know what functionality if any the scoreboard might have with the playoffs?

**Comment by falkyre** [comment]
See this new release:  https://github.com/riffnshred/nhl-led-scoreboard/releases/tag/v1.8.3

**Comment by riffnshred** [comment]
I posted this in the announcement channel on discord yesterday:

IMPORTANT !!! PLEASE READ !!!

Just like at the beginning of the season when we were hit with a new API, we are going into the playoff blind and have no idea what to expect. This mean all playoff features we had wont work this year.  This fix remove key playoff that I hope will prevent crashes and other issues. 

I already started searching for api endpoint and found a few and will keep an eye for others and study how they behave so that I can enable the playoff feature back on for next year. 

Thank you for your understanding and good luck.
